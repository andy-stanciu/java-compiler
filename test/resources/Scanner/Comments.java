/*

this is broken



very broken

/

      /*


 should have /******** many

 problems


/      / /   /     /      /    /

 done.
