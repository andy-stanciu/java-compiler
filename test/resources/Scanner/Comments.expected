DIV TIMES THIS ID(is) ID(broken) ID(very) ID(broken) DIV DIV TIMES ID(should) ID(have) DIV TIMES TIMES TIMES TIMES TIMES TIMES TIMES TIMES ID(many) ID(problems) DIV DIV DIV DIV DIV DIV DIV ID(done) DOT 